### PR TITLE
Update deploy-enterprise-overview.md

### PR DESCRIPTION
**Pull request by Leicham**
modified summary of architecture and misc. copy edits

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/deploy-enterprise-overview.md [hunk 1]
```diff
@@ -23,13 +23,13 @@ appliesto:
...
 
 After you've read the sections below and are ready to start deploying Teams in your organization, see [Set up Microsoft Teams in your enterprise](deploy-enterprise-setup.md).
 
 ## Architecture
 
-Teams is tightly integrated into Microsoft 365 and uses many features to power chat, file sharing, meetings, telephony, video streaming, and more. Before you roll out Teams, check out [Microsoft Teams IT architecture and telephony solutions posters](teams-architecture-solutions-posters.md) to get familiar with how Teams works with the rest of Microsoft 365 to create a complete collaboration experience for your users.
+Teams is tightly integrated into Microsoft 365 and uses many features for collaboration, group chat, private chat, file sharing and live editing, conduct meetings and live events with video capability, integrate telephony to a PSTN, access to hundreds of app integrations, and more. Before you roll out Teams, check out [Microsoft Teams IT architecture and telephony solutions posters](teams-architecture-solutions-posters.md) to get familiar with how Teams works with the rest of Microsoft 365 to create a complete collaboration experience for your users.
```
Comment (JohanFreelancer9) [inline Teams/deploy-enterprise-overview.md:32]:
```suggestion
Teams is tightly integrated into Microsoft 365 and uses many features for collaboration, group chat, private chat, file sharing and live editing, conducting meetings and live events with video capability, integrating telephony to a PSTN, access to hundreds of app integrations, and more. Before you roll out Teams, check out [Microsoft Teams IT architecture and telephony solutions posters](teams-architecture-solutions-posters.md) to get familiar with how Teams works with the rest of Microsoft 365 to create a complete collaboration experience for your users.
```

File: Teams/deploy-enterprise-overview.md [hunk 2]
```diff
@@ -41,9 +41,9 @@ To see deployment planning information for the workload you want to deploy, see
 - [Meetings and conferencing](deploy-meetings-microsoft-teams-landing-page.md)
 - [Phone System and PSTN connectivity](cloud-voice-landing-page.md)
 
-## Network planner
+## Network Planner
 
-Network planning for Teams is extremely important when you have large numbers of users, complex networks, or both. Teams supports voice calling and video conferencing, both of which rely on real-time communications to provide the best experience possible for users. Networks must:
+Network planning for Teams is extremely important when you have any number of users and for all types of networks.  It's essential to ensure your network for 8 users or 8,000 users is capable of delivering the best experience with Teams.  Your existing network infrastructure and provider bandwidth could make a big impact on Teams for your users.  Teams supports voice calling and video conferencing, both of which rely on real-time communications to provide the best experience possible for users. Networks must:
```
Comment (JohanFreelancer9) [inline Teams/deploy-enterprise-overview.md:46]:
```suggestion
Network planning for Teams is extremely important when you have any number of users and for all types of networks.  It is essential to ensure that your network, be it for eight users or 8,000 users, is capable of delivering the best experience with Teams.  Your existing network infrastructure and provider bandwidth could make a big impact on Teams for your users.  Teams supports voice calling and video conferencing, both of which rely on real-time communications to provide the best experience possible for users. Networks must:
```

File: Teams/deploy-enterprise-overview.md [hunk 3]
```diff
@@ -83,12 +83,12 @@ For details about how the Teams advisor works, see [Use Advisor for Teams to hel
 
 As you plan your Teams deployment, you need to consider the lifecycle of teams, channels, files, and so on, in your organization. You should also think about what types of information will be stored in them. Teams may be created for a specific project, for a department, or they might be used as a central resource for the entire organization. Each of these uses have different requirements, which drive questions like the following:
 
-- How long will the teams remain active?
+- How long will the Team remain active?
```
Comment (JohanFreelancer9) [inline Teams/deploy-enterprise-overview.md:86]:
```suggestion
- How long will the team remain active?
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Leicham, kindly commit the suggested changes if you agree with them. Many thanks

**Pull request closed without merging**